packages/ecc-client-elixir-ro-crate/src/components/about/about.ts

```diff
@@ -0,0 +1,445 @@
+
+  static RelatedPeopleFields: Field[] = [
+    {
+      key: "author",
+      label: "Author",
+      type: "group",
+      fieldOptions: {
+        required: false,
+        default: "",
+        tooltip: "The author of this content.",
+      },
+      groupOptions: {
+        collapsible: true,
+      },
+      children: [
+        {
+          key: "person",
+          label: "Person",
+          type: "array",
+          arrayOptions: {
+            defaultInstances: 0,
+          },
+          children: [
+            {
+              key: "personID",
+              label: "@id",
+              type: "url",
+              fieldOptions: {
+                required: true,
+              },
+            },
+            {
+              key: "personType",
+              label: "@type",
+              type: "text",
+              fieldOptions: {
+                default: "Person",
+                required: true,
+              },
+            },
+            {
+              key: "personName",
+              label: "Name",
+              type: "text",
+              fieldOptions: {
+                required: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      key: "publisher",
+      label: "Publisher",
+      type: "group",
+      fieldOptions: {
+        required: false,
+        default: "",
+        tooltip: "The publisher of the creative work.",
+      },
+      groupOptions: {
+        collapsible: true,
+      },
+      children: [
+        {
+          key: "org",
+          label: "Organisation",
+          type: "array",
+          arrayOptions: {
+            defaultInstances: 0,
+          },
+          children: [
+            {
+              key: "orgID",
+              label: "@id",
+              type: "url",
+              fieldOptions: {
+                required: true,
+              },
+            },
+            {
+              key: "orgType",
+              label: "@type",
+              type: "text",
+              fieldOptions: {
+                default: "Person",
+                required: true,
+              },
+            },
+            {
+              key: "orgName",
+              label: "Name",
+              type: "text",
+              fieldOptions: {
+                required: true,
+              },
+            },
+            {
+              key: "orgURL",
+              label: "URL",
+              type: "url",
+              fieldOptions: {
+                required: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      key: "funder",
+      label: "Funder",
+      type: "group",
+      fieldOptions: {
+        required: false,
+        default: "",
+        tooltip:
+          "person or organization that supports (sponsors) something through some kind of financial contribution.",
+      },
+      groupOptions: {
+        collapsible: true,
+      },
+      children: [
+        {
+          key: "org",
```

Comment:
The same issue I stated above applies here, please update the entity to reflect the specification. It is pretty much the same situation as the author and publisher entitties.